### ISSUE TITLE
lowdb conversion fails

### ISSUE THREAD

**Issue by MatthijsKamstra** [haxiomic/dts2hx#65]
Related issue #64

I am sorry about this, probably a very common issue (lib-x doesn't convert...)

So ignore if it's out of the scope.

I was trying to update the documentation I wrote about haxe and node and one of the examples failed when converting.

```
> Converting module @types/lowdb
> Error: Expected variable type but got function (Lowdb [BlockScopedVariable,NamespaceModule] VariableDeclaration ...../03lowdb/code/node_modules/@types/lowdb/index.d.ts:30:13)
> Saved externs for @types/lowdb into .haxelib/lowdb/1,0,9/
> Converting module lodash
> Saved externs for lodash into .haxelib/lodash/4,14,162/
```

Steps to reproduce:

```
npm install dts2hx --save-dev
npm install lowdb 
npm install @types/lowdb
npx dts2hx @types/lowdb
```

**Comment by haxiomic** [comment]
It's good to report lib-x doesn't convert – the aim is for dts2hx to work for all valid typescript definitions

Usually it can continue on when there's an error (but maybe it uses Dynamic or something), do the externs generated for lowdb still work for you?

**Comment by haxiomic** [comment]
This should be fixed now, update to dts2hx 0.14.2 :)

Lowdb uses the function-module pattern, so when you import lowdb in js, you get a function back

To translate this to haxe, because we can't make a module callable, we use a special 'call' field, so you use lowdb like this

```haxe
import Lowdb;

Lowdb.call(...)
```

Rather than
```typescript
let lowdb = require('lowdb')

lowdb(...)
```

**Comment by MatthijsKamstra** [comment]
can confirm that it works, found a new problem...
